lib/improver/orographic_enhancement.py

```diff
@@ -507,6 +523,23 @@ def _create_output_cubes(self, orogenh_data, reference_cube):
         for axis in ['x', 'y']:
             orogenh_standard_grid = sort_coord_in_cube(
                 orogenh_standard_grid, orogenh_standard_grid.coord(axis=axis))
+            orogenh_standard_grid.coord(axis=axis).points = (
+                orogenh_standard_grid.coord(axis=axis).points.astype(
+                    np.float32))
+            if orogenh_standard_grid.coord(axis=axis).bounds is not None:
+                orogenh_standard_grid.coord(axis=axis).bounds = (
+                    orogenh_standard_grid.coord(axis=axis).bounds.astype(
+                        np.float32))
+
+        # add relevant grid definition attributes
```

Comment:
Would you mind putting some example cubes on the ticket to explain the metadata changes you have introduced.

Reply:
I've added details to the ticket.  It might be worth creating some follow on work for this, if we want to refine it.  But I think the changes make this less wrong than it was before!